app/modules/sightings/parameters.py

```diff
@@ -41,6 +41,7 @@ class PatchSightingDetailsParameters(PatchJSONParameters):
         '/locationId',
         '/taxonomies',
         '/verbatimLocality',
+        '/verbatimEventDate',
```

Comment:
@naknomum I added this as it was used in one of the integ tests. Is this correct or should we move the usage from the intreg tests?

Reply:
Might as well leave this as it is present in a lot of data that will be migrated. Even though this is for new AGS we can support it.